src/HostManager.cpp

```diff
@@ -38,39 +38,34 @@
 #include "post_guard.h"
 
 
-bool HostManager::deleteHost( QString hostname )
+bool HostManager::deleteHost(QString hostname)
 {
     qDebug() << "HostManager::deleteHost(" << hostname.toUtf8().constData() << ") INFO: trying to delete host from host pool, getting write lock...";
```

Comment:
I put these debugging lines in but I am finding them just a tad spammy now as things seem to be working OK - is it acceptable to comment out the "INFO" ones - or would it be expected to delete them now and have to put them back if issues crop-up?